### ISSUE TITLE
Make endpoints more RESTful

### ISSUE THREAD

**Issue by broskoski** [AOzone/aozone-kernel#3]
A definite nice-to-have for organization / modeling sake on the median side. 

For instance an account:

`GET /accounts` - (all accounts, not necessarily needed)
`GET /accounts/:id` - (what is currently `/position`)
`POST /accounts` - (what is currently `/create-account`)
`PUT / accounts/:id` - (update an account, would be primarily for depositing funds)
`POST /accounts/:id/buy/:contract_id` - (takes number of shares)
`POST /accounts/:id/sell/:contract_id` - (takes number of shares)

or contracts:

`GET /contracts` - (all contracts)
`GET /contract/:id` - (data for a single contract, with buy and sell quotes including historical charts, an optional parameter could determine the scope of a chart)

Let me know if this is a pain or unclear.

**Comment by dochugo** [comment]
Hmm, I usually default to GET for all commands in non-sensitive apps
because POST/PUT are a pain to debug via browser.  But if your endpoint
requires it, I can certainly add support for those METHODS in addition to
GET

**Comment by broskoski** [comment]
Yeah, that will work, feel free to leave GET routes for yourself to debug but for anything that is modifying data, GET is so non-standard that it gets difficult to model on the API consumer end.

I usually use https://chrome.google.com/webstore/detail/advanced-rest-client/hgmloofddffdnphfgcellkdfbfbjeloo?hl=en-US for debugging via the browser (which supports all HTTP methods, headers, etc).

For reference:
http://www.vinaysahni.com/best-practices-for-a-pragmatic-restful-api#restful
and http://backbonejs.org/#Sync (what I am using to model on my side)
